### PR TITLE
Fix/ingredient bug

### PR DESCRIPTION
**Pull request by FrazierLE**
Summary of the work done:

- This completes bug fixes with ingredients list and saved recipe page.

This branch includes:

- Bug fixes with ingredients list and saved recipe page by recipe class

Contributed to code:

- Name Lauren & Eleanor

Needs to review:

- Name: Kiko & Jenn

### CODE REVIEW

File: src/classes/Recipe.js [hunk 1]
```diff
@@ -15,12 +15,15 @@ class Recipe {
     determineIngredients(ingredientInfo) {
         this.ingredientsList = ingredientInfo.reduce((acc, ingredient) => {
             this.ingredients.forEach((item) => {
-                if (item.id === ingredient.id)
-                  {acc.push(ingredient.name)};
+              if (item.id === ingredient.id) {
+                let name = ingredient.name;
+                let list = {ingredient: `${item.quantity.amount.toFixed(2)} ${item.quantity.unit} ${name}`}
+                acc.push(list)
+                }
```
Comment (jmyacobn) [inline src/classes/Recipe.js:22]: Thank you for fixing our ingredients list issue!!!

File: src/scripts.js [hunk 2]
```diff
@@ -98,28 +97,11 @@ function viewRecipeIngredients(event) {
...
-  ingredientsArray.forEach(curr => {
-    quant = `${curr.quantity.amount.toFixed(2)}`
-    unit = `${curr.quantity.unit}`
-    amount = "<p>" + quant + " " + unit + "</p>"
-    amountArray.push(amount);
-    ingredientListAmounts = `${amountArray.join(" ")}`
+  let listOfIngredients = foundRecipe.determineIngredients(ingredientsData);
+  ingredientList.innerHTML = ''
+  listOfIngredients.forEach((item) => {
+        ingredientList.innerHTML += `<p>${item.ingredient}</p>`;
```
Comment (jmyacobn) [inline src/scripts.js:103]: Thank you for pulling in from the class to fix our bandaid!

**Review by knpausch — APPROVED** [review]
great! thanks for fixing our ingredient quantity issue

**Review by jmyacobn — APPROVED** [review]
This looks great and fixed one of our big issues! Thank you!

**Pull request merged**